integration/v7/isolated/service_command_test.go

```diff
@@ -215,6 +218,52 @@ var _ = Describe("service command", func() {
 					))
 				})
 			})
+
+			When("the instance is shared with another space", func() {
+				var sharedToSpaceName string
+
+				BeforeEach(func() {
+					sharedToSpaceName = helpers.NewSpaceName()
+					helpers.CreateSpace(sharedToSpaceName)
+
+					output := func() *Buffer {
```

Comment:
It took me a while to work out why this is here. Perhaps we should hide the asynchronicity of the create in a helper function.

Reply:
Yea this makes sense, it was the last thing I wrote in this commit as I realized the service wasn't being shared (and silently succeeding) when the service had not finished being created.  I wasn't sure the right way to break out helper methods, but could see something along the lines of `createAndWaitForService` or something.